app/views/policies/new.html.slim

```diff
@@ -7,4 +7,4 @@
             | Add Policy
           .clearfix
         = render partial: 'form'
-
+= javascript_include_tag 'theme'
```

Comment:
@AmiFaldu-BTC : Can we include this js in common.js as it should be common to all pages?